Sem. 09/Task2.cpp

```diff
@@ -0,0 +1,43 @@
+//Задача 2 Напишете функции toUpper и toLower, които приемат стринг и променят всички главни букви в малки/всички малки букви в главни.
```

Comment:
In the provided problem we meant converting all uppercase letters to lowercase and vice-versa simultaneously i.e

**Example:**

```c++
"HeLlo WoRLd"
```

```c++
hElLO wOrlD
```